service/src/main/java/bio/terra/workspace/service/workspace/exceptions/MissingSpendProfileException.java

```diff
@@ -8,8 +8,12 @@
  * profile associated with it.
  */
 public class MissingSpendProfileException extends BadRequestException {
-  public MissingSpendProfileException(UUID workspaceId) {
-    super(
+  public MissingSpendProfileException(String message) {
+    super(message);
+  }
+
+  public static MissingSpendProfileException missingSpendProfileException(UUID workspaceId) {
```

Comment:
nit: Perhaps `forWorkspace` or something else? I don't think naming a factory method the same as the class adds much information.